src/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -456,196 +423,242 @@ public static Player checkPlayer(CommandSource source) throws CommandException {
-                .permission("griefprevention.deleteclaims")
-                .executor(new CommandDeleteClaim())
+                .permission("griefprevention.dcommand.claim.delete")
+                .executor(new CommandClaimDelete())
+                .build());
+        
+        claimSubcommands.put(Arrays.asList("transfer", "give"), CommandSpec.builder()
+                .description(Texts.of("Converts an administrative claim to a private claim"))
+                .arguments(optional(player(Texts.of("target"))))
+                .permission("griefprevention.command.claim.transfer")
+                .executor(new CommandClaimTransfer())
                 .build());
         
-        subcommands.put(Arrays.asList("claimexplosions"), CommandSpec.builder()
+        claimSubcommands.put(Arrays.asList("explosions"), CommandSpec.builder()
                 .description(Texts.of("Toggles whether explosives may be used in a specific land claim"))
-                .permission("griefprevention.claims")
+                .permission("griefprevention.command.claim.explosions")
                 .executor(new CommandClaimExplosions())
                 .build());
         
-        subcommands.put(Arrays.asList("deleteallclaims"), CommandSpec.builder()
+        claimSubcommands.put(Arrays.asList("deleteall"), CommandSpec.builder()
                 .description(Texts.of("Delete all of another player's claims"))
-                .permission("griefprevention.deleteclaims")
+                .permission("griefprevention.command.claim.deleteall")
                 .arguments(player(Texts.of("player"))) // TODO: Use user commandelement when added
-                .executor(new CommandDeleteAllClaims())
+                .executor(new CommandClaimDeleteAll())
                 .build());
 
-        subcommands.put(Arrays.asList("claimbook"), CommandSpec.builder()
+        claimSubcommands.put(Arrays.asList("book"), CommandSpec.builder()
                 .description(Texts.of("Gives a player a manual about claiming land"))
-                .permission("griefprevention.claimbook")
+                .permission("griefprevention.command.claim.book")
                 .arguments(playerOrSource(Texts.of("player")))
                 .executor(new CommandClaimBook())
                 .build());
 
-        subcommands.put(Arrays.asList("claimslist", "claimlist", "listclaims"), CommandSpec.builder()
+        claimSubcommands.put(Arrays.asList("list"), CommandSpec.builder()
                 .description(Texts.of("List information about a player's claim blocks and claims"))
+                .permission("griefprevention.command.claim.list")
                 .arguments(onlyOne(playerOrSource(Texts.of("player"))))
-                .executor(new CommandClaimsList())
+                .executor(new CommandClaimList())
+                .build());
+        
+        claimSubcommands.put(Arrays.asList("flag"), CommandSpec.builder()
```

Comment:
IT would be better to make the claim subcommands part of the CommandClaim.getCommand() or something.